.github/workflows/ci.yml

```diff
@@ -44,7 +44,7 @@ jobs:
         uses: FreeRTOS/CI-CD-Github-Actions/complexity@main
         with:
           path: ./
-          horrid_threshold: 12
```

Comment:
to be frank here - I did not want to spend extra time reducing the complexity :)